aspnetcore/includes/scaffold-identity/migrations.md

```diff
@@ -0,0 +1,21 @@
+The generated Identity database code requires [Entity Framework Core Migrations](/ef/core/managing-schemas/migrations/). Create a migration and update the database. For example, run the following commands:
+
+# [Visual Studio](#tab/visual-studio) 
+
+In the Visual Studio **Package Manager Console**:
+
+```PMC
+Add-Migration CreateIdentitySchema
+Update-Database
+```
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+```cli
+dotnet ef migrations add CreateIdentitySchema
+dotnet ef database update
+```
+
+------
+
+The `Add-Migration` name is arbitrary, "CreateIdentitySchema" describes the migration.
```

Comment:
On first read, this makes me thing that `Add-Migration` itself is arbitrary, but Add-Migration is the name of the command. Maybe something like this would be clearer:
`When running the Add-Migration command, the parameter name "CreateIdentitySchema" is arbitrary, any value can be provided. It's meant to describe the migration.`